php-nginx/Dockerfile

```diff
@@ -16,9 +16,6 @@
 
 FROM gcr.io/google_appengine/debian8
 
-# Cache busting for security update
-ENV UPDATED_AT 2016-11-15
```

Comment:
Cloud Container Builder service that we use for building the image, used to run `Docker build` without `--no-cache` option, so we used to need this for cache busting. Now the Cloud Container Builder Service is using '--no-cache' option, so we don't need this any more.

Re: apt-get updates

Yeah we're running apt-get update && apt-get upgrade just below.